src/refactor_nrepl/ns/tracker.clj

```diff
@@ -55,7 +55,8 @@
   (let [deps (dep/immediate-dependents (:clojure.tools.namespace.track/deps tracker)
                                        (symbol my-ns))]
     (for [[file ns] (:clojure.tools.namespace.file/filemap tracker)
-          :when ((set deps) ns)]
+          :when ((set deps) ns)
+          :when (not (util/interrupted?))]
```

Comment:
Do we need this?  This final filtering of files using a set membership should be blazing fast.

Reply:
Note that we're inside a `for` so `((set deps) ns)` will be invoked many times. So by observing cancellation we make the `for` interruptible.

Still, thanks for the scrutiny. I changed two things now:

* Calculate `(set deps)` just once
* Invoke `util/interrupted?` _prior_ to the set membership query - I'd expect the former to be faster.